apps/pr-deploy-site/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <body>
```

Comment:
It might be nice to add a `<title>` for folks using assistive technology while reviewing.

I don't anticipate many folks bookmarking individual PR deploy sites.